### DISCUSSION TITLE
Hola, como pedir un reembolso?

### DISCUSSION THREAD

**Discussion by Darkness036** [aclap-dev/video-downloadhelper#768] [Help and Q&A]
### Your question

Quiero pedir un reembolso porque como lo veo la extensión no sirve para nada, es más, las que son gratuitos sirven

### Addon & CoApp version and details

Quiero pedir un reembolso porque como lo veo la extensión no sirve para nada, es más, las que son gratuitos sirven

**Comment by jcfrog** [comment]
send us a mail at licenses@downloadhelper.net
